tpm2tools/flush_active_handles_test.go

```diff
@@ -0,0 +1,100 @@
+package tpm2tools
+
+import (
+    "crypto/rand"
+    "crypto/rsa"
+    "testing"
+
+    "github.com/google/go-tpm/tpm2"
+
+    "github.com/google/go-tpm-tools/simulator"
+)
+
+const (
+    // How many keys/handles can the simulator contain at once.
+    maxHandles = 3
+)
+
+func loadRandomExternalKey(t *testing.T, simulator *simulator.Simulator) {
+    pk, err := rsa.GenerateKey(rand.Reader, 2048)
+    if err != nil {
+        t.Fatal(err)
+    }
+    public := tpm2.Public{
```

Comment:
If possible copy then modify one of the templates already in this package, rather then rewriting all of this again

Reply:
I've moved this into its own testing helper under /tpm2tools/tpm2toolstest/